pubspec.yaml

```diff
@@ -0,0 +1,6 @@
+name: wCommon
+description: Dart bindings for wf-common
+dependencies:
+dev_dependencies:
+  unittest: '>=0.11.0 <0.11.6'
+  mockito: '>=0.8.2'
```

Comment:
Use this for mockito:

```
  mockito: ^0.8.2
```

Which is equivalent to `>=0.8.2 <0.9.0` since it's less than 1.0.0, right?